src/index.js

```diff
@@ -1 +1,53 @@
-import './index.css'
\ No newline at end of file
+import './index.css';
+
+const tasks = [
+  {
+    description: 'Wash Dishes',
+    completed: 'false',
```

Comment:
- [ ] As mentioned in the project requirements :point_down: please make sure the completed attribute has boolean value either true or false, currently you have mistakenly assigned false as a string value, please fix this by assigning a boolean either true or false.
![image](https://user-images.githubusercontent.com/97521557/221349322-95bb4551-5208-415c-ad87-d981d8fb2da0.png)